src/containers/SideBar/SideBar.css

```diff
@@ -0,0 +1,48 @@
+.CategoriesSideBar{
+    vertical-align: text-top;
+}
+
+.SideBarItem{
```

Comment:
Let's add the appropiate `cursor:pointer` to indicate this is a clickable element